x/evm/keeper/evm.go

```diff
@@ -118,6 +118,23 @@ func (k *Keeper) CallEVM(ctx sdk.Context, from common.Address, to *common.Addres
 	if res.Err != nil {
 		vmErr = res.Err.Error()
 	}
+	existingReceipt, err := k.GetTransientReceipt(ctx, ctx.TxSum())
```

Comment:
is this logic related to the wasmd precompile stuff?

Reply:
yeah it's to merge with the temp receipts created in the CW->EVM hop